roles/openshift/tasks/firewalld.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
@detiber so with sdn, couldn't that be a role too?

I envision this:
openshift-master
openshift-node
openshift-router (if needed)
openshift-sdn
openshift-gce-lb (configure a gce load balancer for the router)
etc.

As you can see, I imagine having a lot of roles for OpenShift.

I'm not a fan of 'wrapper roles' where we keep the all in one role and then just wrap it. That seems like overkill and extra work to manage, without giving any benefit.

I'd prefer openshift-common as a base dependency to having multiple roles all inside of a single role, but there would have to be quite a bit of duplication for this to be worthwhile IMO.

Specific role ordering will be something necessary, but I'd prefer to not combine all of the ansible roles into a single role (with variable selector) to solve this. I think we can solve that with example playbooks, or even by generating the playbooks ourselves like what oo-install in v2 does with puppet.